contributor-growth/drafts/contributor-growth-framework/1-Motivating.md

```diff
@@ -0,0 +1,107 @@
+   * Want to contribute to a project, whether to gain experience or just for fun, and pick one;
+   * Want to learn how to write code in a language (for language projects).  
+
+As a project maintainer, these "trigger motivators" are mostly out of your control. What you can control is providing users with the tools they need to follow through and get them excited about the role they can potentially play. Focus on:
+   1. Communication: Be really clear about how they can participate, what you need, and what's in for them;
+   2. Workflow: Ensure the process is as intuitive and frictionless as possible.
+
+Each additional step they must take -- whether seeking info on how to contribute or submitting a PR -- represents a friction point that may demotivate them and lead to a drop-off. 
+Another less tangible but equally, if not more, important piece is the human factor. Contributors want to feel valued and belong. Creating a welcoming community is key. 
+
+## Honest and Clear Communication 
+
+The first thing prospective contributors will want to know is the level of effort it will take on their end and what they will gain -- be clear about that. This will ideally be captured in your contributor ladder (you can find a contributor ladder template here). 
+
+Shared ownership and governance is also a huge motivator. The prospect of molding the project and playing a role in its development can be a huge intrinsic motivator. Be vocal about it. When you bring in people, tell them that you're willing to share responsibility. All too often maintainers don't spell it out, assuming contributors know. This is especially important early on in the project.
+
+Be open about your roadmap and what you are trying to accomplish. Contributors want to be inspired. After all, you're asking them to dedicate their time and effort to your project. Be transparent about your vision and how you're planning on executing it. One important aspect about roadmap transparency is to let contributors know that it may change as the project matures. 
+
+When you discuss the roadmap, make sure it's all discussed publicly (e.g. community meetings, Slack, discourse, mailing list, GitHub issues or discussions). This will also help avoid situations in which someone submits a massive PR out of the blue which will almost certainly end up in a bad contributor experience. PRs aren't the best place to have feature discussion, so make sure you provide a platform for that discussion to happen (e.g Slack, GitHub Discussions, etc.). 
+
+Be clear and descriptive about how people can get involved and document it well. Good "first issues" and "help wanted" labels are great ways to signal how people can get started. It also shows that the maintainers have taken the time to identify how newcomers can help.  But also be explicit about what you need and communicate it where your users are (e.g. Twitter, Slack, tech blogs, docs, releases).  
```

Comment:
Not a blocker but it would be great to link to our guidance on curating your good first issues

https://github.com/cncf/sig-contributor-strategy/blob/master/contributor-growth/docs/issue-labels.md